src/src/AddBookForm/AddBookForm.tsx

```diff
@@ -0,0 +1,5 @@
+import * as React from 'react'
```

Comment:
Why do you import `* as`?  I think that `import React from 'react'` will do the job.

Reply:
You are right. In the past, I had an error when trying to import the way you suggested. But now I'm using a new tsconfig file, there is "allowSyntheticDefaultImports" rule which allows importing normal way. I corrected it in my files.